product/reports/170_Configuration Management - Containers/130_Images by Failed Openscap Rule Results.yaml

```
@@ -0,0 +1,64 @@
+---
+title: Images by Failed Openscap Rule Results
+rpt_group: Custom
+rpt_type: Custom
+priority:
+db: ContainerImage
+cols:
+- name
+include:
+  openscap_result:
+    include:
+      openscap_rule_results:
+        columns:
+        - name
+        - result
+        - severity
+col_order:
+- name
+- openscap_result.openscap_rule_results.name
+- openscap_result.openscap_rule_results.result
+- openscap_result.openscap_rule_results.severity
+headers:
+- Container Image Name
+- Rule Name
+- Result
+- Severity
+conditions: !ruby/object:MiqExpression
+  exp:
+    CONTAINS:
+      field: ContainerImage.openscap_result.openscap_rule_results-result
```

Does this work? MiqExpression should raise an error on parsing this field